app/templates/promise-tree.hbs

```diff
@@ -9,18 +9,18 @@
     @schema={{schema-for "promise-tree"}}
     @setIsDragging={{action "setIsDragging" target=this.application}} as |list|
   >
-    {{#list.vertical-collection filtered as |content index|}}
+    {{#list.vertical-collection this.filtered as |content index|}}
       <tr class="list__row js-promise-tree-item {{if (mod index 2) "striped"}}">
-        {{promise-item
-          model=content
-          filter=filter
-          effectiveSearch=effectiveSearch
-          toggleExpand=(action "toggleExpand")
-          tracePromise=(action "tracePromise")
-          inspectObject=(action "inspectObject")
-          sendValueToConsole=(action "sendValueToConsole")
-          list=list
-        }}
+        <PromiseItem
+          @model={{content}}
+          @filter={{this.filter}}
+          @effectiveSearch={{this.effectiveSearch}}
+          @toggleExpand={{fn this.toggleExpand content}}
```

Comment:
I'm curious why only two of the actions needed `content` passed?

Reply:
I like to keep the components as dumb as possible but maybe consistency should win here.

Reply:
If only two need it, that's fine. I just wanted some clarity.